src/components/BitcoinWallet.vue

```diff
@@ -183,14 +183,32 @@
                       <span
                         v-if="
                           tx.description === 'Lightning Wallet' &&
-                            tx.type === 'outgoing'
+                            tx.type === 'outgoing' &&
+                            tx.confirmations === 0
+                        "
+                        >&bull; Channel opening</span
+                      >
+                      <span
+                        v-else-if="
+                          tx.description === 'Lightning Wallet' &&
+                            tx.type === 'outgoing' &&
+                            tx.confirmations > 0
                         "
                         >&bull; Channel open</span
                       >
```

Comment:
```suggestion
                       <span
                         v-if="
                           tx.description === 'Lightning Wallet' &&
                           tx.type === 'outgoing' 
                        "
                        >&bull; {{ tx.confirmations === 0 ? "Channel opening" : "Channel open" }}</span
                      >
```
Maybe you can have only one span and change the text depending on the confirmation condition instead of having two identical span?

Reply:
Yeah totally, my bad I've done it to fast, I will update it and do the same for closing